### DISCUSSION TITLE
Update to Postal V3 issue

### DISCUSSION THREAD

**Discussion by kayastha1988** [postalserver/postal#2944] [Help with using Postal]
I have upgraded my postal SMTP from v2 to v3. All went smoothly, no issue warning error during update. Now my SMTP server is not working. Have made changes on postal.yml file as per new configuration, but still my SMTP is not working. How can I solve this issue?

**Comment by willpower232** [comment]
When you say it isn't working, have you checked `postal status`? You might need to try `postal logs smtp` to see what is happening.

**Reply by kayastha1988** [reply]
Hello just a quick question. Postal is running and work so far good. Emails to gmail is going in spam and says SPF failed. I have added gmail spf record in my DNS. Do I missing something else so that SPF pass and email goes to inbox instead spam?
**include:_spf.google.com**

Looking forward to hearing from you.

Thanks
